### ISSUE TITLE
Babashka integration?

### ISSUE THREAD

**Issue by Olical** [clojure-vim/vim-jack-in#5]
How about `:Bb` or `:Babashka` (I'm open to suggestions here) to start up a bb process with an nREPL on a random port with the port written to `.nrepl-port`? Then all Vim / Neovim Clojure tooling can just hook into that seamlessly :smile:

(I am planning on doing this but thought I'd put it out in the open first to get feedback)

**Comment by SevereOverfl0w** [comment]
I've been thinking about this too.  Although one problem is that there's no middleware right now, so the biggest value (plugins providing the list of middleware) is not provided.  But it certainly stops you from needing to know some details.

**Comment by Olical** [comment]
Yeah, good point, the middleware is great, maybe that's something that could be added (one day in the future if `bb` ever supported something extra and injectable like this). One thing this could provide over regular `bb` in a shell is that it picks a good nREPL port for you and writes it to a file. IIRC The Borkmeister himself decided not to do those parts which is completely fair, it leaves it up to you which is great. We could provide those "sensible" (in this context) defaults for seamless Clojure + (Neo)vim plugin integration.
